tools/platforms/MacPlatform.hx

```diff
@@ -535,6 +535,6 @@ class MacPlatform extends PlatformTarget
 
 	private inline function get_dirSuffix():String
 	{
-		return targetArchitecture == X64 ? "64" : "";
+		return targetArchitecture == X64 ? "64" : targetArchitecture == ARM64 ? "Arm64" : "";
```

Comment:
The ndll directory should be `MacArm64`. However, I'm not sure if this makes sense everywhere where `dirSuffix` is used, for example, the build directory would be `macArm64`, which looks odd with the upper case letter.

Reply:
Addressed in 4ed893afbd3a9b16990b530fd3fcaca679fb6d12.

Reply:
Thank you!